linera-chain/src/block_tracker.rs

```diff
@@ -351,6 +351,15 @@ impl<'resources, 'blobs> BlockExecutionTracker<'resources, 'blobs> {
             .collect()
     }
 
+    /// Returns stream ids for events published in the block.
```

Comment:
```suggestion
    /// Returns stream IDs for events published in the block.
```

(Should also be "IDs" in the other comments.)